ray-operator/controllers/ray/utils/httpproxy_httpclient.go

```diff
@@ -29,8 +29,8 @@ type RayHttpProxyClient struct {
 	client       *http.Client
 	httpProxyURL string
 
-	mgr      ctrl.Manager
-	useProxy bool
+	mgr                ctrl.Manager
+	UseKubernetesProxy bool
```

Comment:
Why do we need to export this field?

Reply:
In Go, an upper case field is a public exported. We can use the lower case useKubernetesProxy to make it private.

Reply:
Thanks to @rueian  for the illustration! I have made it private.